package/src/handler.test.ts

```diff
@@ -74,3 +80,228 @@ describe('search requests', () => {
         }
     })
 })
+
+describe('docstrings', () => {
+    it('finds nothing when no comment style is specified', async () => {
+        assert.deepStrictEqual(
+            findDocstring({
+                fileText: `
+        def foo():
+            """docstring"""
+            pass
+        `,
+                definitionLine: 1,
+            }),
+            undefined
+        )
+    })
+
+    it('finds one-line python doc block', async () => {
+        assert.deepStrictEqual(
+            findDocstring({
+                fileText: `
+        def foo():
+            """docstring"""
+            pass
+        `,
+                definitionLine: 1,
+                commentStyle: pythonStyle,
+            }),
+            'docstring'
+        )
+    })
+
+    it('finds multi-line python doc block', async () => {
```

Comment:
All of these tests are `async` functions, but the test body seems sync. Am I missing something?

Reply:
I copy-pasted, so no you're not missing anything 😄